data/json/recipes/food/baking.json

```
@@ -65,5 +65,29 @@
       [ [ "salt", 2 ] ],
       [ [ "cinnamon", 2 ] ]
     ]
+  },
+  {
+    "type": "recipe",
+    "result": "baklava",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_SNACK",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "2 hours",
+    "batch_time_factors": [ 80, 1 ],
+    "charges": 14,
```

I'm a dumb, I adjusted the wrong way. 
```suggestion
    "charges": 7,
```